plugins/destiny.py

```diff
@@ -343,6 +346,20 @@ def triumph(text, nick, bot):
 
 @hook.command('xur')
 def xur(text, bot):
+    now = datetime.datetime.now(DAILY_RESET_TIMEZONE)
+    weekday = now.weekday()
+    # xur is available from friday's reset until sunday's reset, i.e. friday (4) and saturday (5)
+    xursday_diff = 4 - weekday
+    if xursday_diff < -1: # if past saturday, bump to next week
+        xursday_diff += 7
+
+    xursday = (now + timedelta(days=xursday_diff)).replace(hour=0, minute=0, second=0, microsecond=0)
+
+    time_to_xursday = xursday - now
+
+    if time_to_xursday > datetime.timedelta(0):
+        return '\x02Xûr will return in\x02 {}'.format(time_to_xursday.replace(microsecond=0))
```

Comment:
this will produce an error. time_to_xurday doesn't have a replace method. Also, the output isn't the best.